src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: 0;
+  font-family: $main-font;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card {
+  padding: 16px;
+  padding-top: 32px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: $main-white-color;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__image {
+  box-sizing: border-box;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__image img {
+  width: 100%;
+  height: 100%;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  align-self: flex-start;
+  color: $main-color-text;
```

Comment:
Ensure that the variable `$main-color-text` is defined. If it's not defined, it will cause a compilation error.